### ISSUE TITLE
Choose the hour for snapshot.

### ISSUE THREAD

**Issue by sebastienserre** [linuxmint/timeshift#115]
Hello,

In the schedule tab, should it be possible to choose at which moment of the day the snapshot is done ?
This could be a date and time but also "at boot", daily, hourly, monthly....

Regards

**Comment by Blueshawk** [comment]
I second this. It would be a nice enhancement.

**Comment by BradB58** [comment]
Yes, I agree 100%! I do not have a slow computer and yet I can't seem to do much at all when Timeshift starts running. There also seems to be no way to pause a running backup. Because of this inconvenience, I had to turn off scheduled backups. I just have to remember to run Timeshift manually before kernel or driver updates. Recently I forgot, an unfortunately I ended up with some frustration which could have been avoided if I had had a more recent snapshot. So anyway, here's the deal; I would like to know why no one seems interested in implementing fixed scheduling. There have been plenty of people asking for this feature. Timeshift as I understand, was designed to restore your system and not really recommended for backing up personal data. The majority of users do not need Timeshift to be constantly running backups every hour at what ever time it wishes. Once a day, should be more than enough IMO. There is a three hour period in the day when I leave my computer running unattended. This is the time that I schedule my backups, and I sure would like to include Timeshift in that schedule. I use the Rsync GUI Luckybackup for my user data and it allows me to easily schedule a cron job for my backup from right within the program. Please someone explain to me why this would be so much of a problem to implement into Timeshift? Are we really asking too much?

**Comment by cscullion** [comment]
This feature would be very nice for me also.  It's possible to force this behavior by manually modifying the cron file (/etc/cron.d/timeshift-hourly).  However, timeshift will overwrite your changes after each run.  The only way I've found to prevent this is to make that file immutable: 

`sudo chattr +i /etc/cron.d/timeshift-hourly`

This does yield an error in the log when timeshift tries to overwrite the existing file, but I haven't noticed any other unwanted side effects.

I need this because I don't like leaving my laptop on overnight, and you really can't shutdown the laptop if a timeshift backup started just before you're ready to shut down for the day.  So, I set my cron file to "0 6-20 * * *" so backups only start between 6am and 8pm.

**Comment by dmolin** [comment]
+1 on this feature.
Timeshift decides automatically when to do the daily backup and this is not great since it's outside of my control and I cannot guarantee that my computer will always be turne on at that time. On the weekend, as an example, if my laptop is switched off at the time Timeshift "decided" to back it up, this will result in Timeshift arbitrarily changing the backup time at the next rebook, making this process constantly unpredictable
